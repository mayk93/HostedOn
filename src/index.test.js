/**
 * Created by michael on 06/02/2018.
 */

import React from 'react';
import renderer from 'react-test-renderer';

import HostedOn from './index';

test('HostedOn has expected structure', () => {
  const component = renderer.create(
    <HostedOn />,
  );

  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();

  console.log('Tree:');
  console.log(tree);

  expect(tree.type).toEqual("div");
  expect(tree.children.length).toEqual(1);
  expect(tree.children[0].type).toEqual("div");
});