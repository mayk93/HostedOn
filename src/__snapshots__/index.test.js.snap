// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HostedOn has expected structure 1`] = `
<div>
  <div
    style={
      Object {
        "backgroundColor": "#F8F8F8",
        "borderTop": "1px solid #E7E7E7",
        "bottom": "0",
        "height": "25px",
        "left": "0",
        "padding": "20px",
        "position": "fixed",
        "textAlign": "center",
        "width": "100%",
        "zIndex": 10,
      }
    }
  >
    <div
      style={
        Object {
          "cursor": "pointer",
          "flex": 1,
        }
      }
      title="https://en.wikipedia.org/wiki/Internet"
    >
      Hosted on
      <div
        onClick={[Function]}
        style={
          Object {
            "color": "black",
            "display": "inline-flex",
          }
        }
      >
        <span
          aria-hidden={true}
          className="fa fa-cloud fa-2x"
          style={
            Object {
              "marginLeft": "5px",
            }
          }
        />
      </div>
    </div>
  </div>
</div>
`;
